setup.py

```diff
@@ -39,6 +39,7 @@
         'ctaplot~=0.5.5',
         'gammapy~=0.19.0',
         'uproot~=4.1',
+        'numpy<1.22.0a0',
```

Comment:
I think it would be better to have the same condition in environment and setup.py, because now they ask for 1.21 or for <1.22.0a0 (both can be fulfilled simultaneously obviously, but why to limit it more in one place than in the other?

Reply:
and yes, I had to make some fixes on the CI because it gave some errors that before it did not give (e.g. when merging #141 ).